Godeps/_workspace/src/github.com/spf13/cobra/bash_completions.md

```diff
@@ -67,7 +67,7 @@ cmds := &cobra.Command{
 	Short: "kubectl controls the Kubernetes cluster manager",
 	Long: `kubectl controls the Kubernetes cluster manager.
 
-Find more information at https://github.com/GoogleCloudPlatform/kubernetes.`,
+Find more information at https://github.com/kubernetes/kubernetes.`,
```

Comment:
This is going to break a precommit check (travis will be red). We should leave Godeps/ as is.